samples/subsys/tracing/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _tracing_sample:
```

Comment:
please make sure you actually rename the existing README.txt -- right now it's not showing up in your commit whereas we'd want it to be gone :)